frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -50,13 +51,7 @@ import type { Ref } from 'vue';
 import type { AutoCompleteCompleteEvent } from 'primevue/autocomplete';
 import type { DropdownChangeEvent } from 'primevue/dropdown';
 import type { IInputEvent } from '@/interfaces';
-import type { Permit } from '@/types';
-
-// Types
-type GeocoderEntry = {
-  geometry: { coordinates: Array<number>; [key: string]: any };
-  properties: { [key: string]: string };
-};
+import type { GeocoderEntry, Permit } from '@/types';
```

Comment:
I'm not seeing this type used outside of this file. No harm in keeping it local at this point.